crowdin.yml

```
@@ -0,0 +1,14 @@
+'project_id': '813672'
+'api_token': '${CROWDIN_API_TOKEN}'
```

If we want to use their build-in support for env vars.

```suggestion
'api_token': '${CROWDIN_API_TOKEN}'
```